sdk/eventhub/azure-eventhub/azure/eventhub/_consumer_client.py

```diff
@@ -107,7 +122,13 @@ def __init__(
     ):
         # type: (...) -> None
         self._checkpoint_store = kwargs.pop("checkpoint_store", None)
-        self._load_balancing_interval = kwargs.pop("load_balancing_interval", 10)
+        self._load_balancing_interval = kwargs.pop("load_balancing_interval", None) or 10
```

Comment:
the difference vs. the former is subtle here, namely that now if they provide "0" as load_balancing_interval, it would become "10."

was this the intention?  I wonder if throwing a ValueError is more appropriate if that's the expectation so that the user doesn't think they're handing 0 then get a totally different behavior.

Reply:
(same comment below with the ownership expiration interval)

Reply:
I updated to convert 'None' to default value and allow 0 to pass without converting to default value. 
0 isn't a good value. I don't think a user will use it. But if a user really uses it, changing to raise ValueError would be a break change.

Reply:
Ok, I see.  My primary goal was making sure that we didn't change how we interpreted user settings unexpectedly; the fact that it's a non-sensical input is a good callout though.  

I might respond to that by saying "whatever the situation, users would have hit that in the old world comparable to the new, and if we run into problems with that we'd likely triage it the same way by making an explicit valueError?"
(More tersely, hopefully we're still fine since, at worst, we have the same behavior we used to in this respect)